README.md

```diff
@@ -1,2 +1,78 @@
 # Tao dependency resolver
 
+Resolves the requires tree from manifests.php in each extension needed.
+
+/!\ Works on **extension** names, not repository names.
+
+A more extensive explanation of the problematics and solutions is exposed in the [documentation](doc/dependency-resolver.md).
+
+
+## Installation
+
+Clone this repository.
+
+Install dependencies :
+
+```
+$ composer install
+```
+
+Minimal PHP version required: 7.1
+
+PHP extensions required: php7.1-xml, php7.1-mbstring
+
+
+## Authentification
+
+You need to provide a valid [GitHub token](https://github.com/settings/tokens) with "repo" access rights into `<project dir>/config/services.yaml`, in the key `parameters > github.token`.
+
+
+## The tools
+
+There are two tools in this repository:
+
+### Dependency resolver
+
+Read more about this tool [here](doc/dependency-resolver.md).
+
+```
+$ php bin/console dependencies:resolve <root extension or repository name> [--package-branch <root extension branch>] [--extension-branch <dependency extensions branch>] [--directory <directory>] 
```

Comment:
the parameter `--extension-branch` should be `--extensions-branch`

Reply:
Fixed